ml-agents/tests/yamato/run_script_in_venv.py

```diff
@@ -0,0 +1,35 @@
+import sys
+import subprocess
+
+from .yamato_utils import get_base_path, init_venv
+
+
+def run_script_in_venv(script_path):
+    """
+    Sets up the venv, runs the specified script, and returns based on whether the script succeeded.
+    :return:
+    """
+    base_path = get_base_path()
+    print(f"Running in base path {base_path}")
+
+    venv_path = init_venv()
+
+    res = subprocess.run(
+        f"source {venv_path}/bin/activate; python {script_path}", shell=True
+    )
+
+    if res.returncode != 0:
+        print(f"{script_path} run FAILED!")
+        sys.exit(1)
+
+    print("{script_path} run SUCCEEDED!")
+    sys.exit(0)
```

Comment:
```suggestion
    if res.returncode != 0:
        print(f"{script_path} run FAILED!")
        sys.exit(1)
    else:
        print("{script_path} run SUCCEEDED!")
        sys.exit(0)
```
Is this easier to read ?

Reply:
is the `else` needed in your suggested change? I was under the assumption that the program would exit after the call to `exit`

Reply:
It's going to be a moot point when I move the venv setup to its own command

Reply:
Both scripts now take an `--env` commandline option (same syntax as `mlagents-learn`), which defaults to `Project/testPlayer`